Documentation: FAQ refers to aor-dependent-input

aor-dependent-input does not seem to work with the current version of react-admin and leads into an error:

> Module not found: Can't resolve 'admin-on-rest/lib/mui/form/FormField' in '<my local project path>\node_modules\aor-dependent-input\lib'

The problem: aor-dependent-input is mentioned on the FAQ page https://marmelab.com/react-admin/FAQ.html. What can be used instead  aor-dependent-input?

--- a/docs/FAQ.md
+++ b/docs/FAQ.md
@@ -148,7 +148,7 @@ Some use cases:
 - Show/hide some inputs if another input has a specific value
 - Show/hide some inputs if the current form value matches specific constraints
 
-For all those cases, you can use the [aor-dependent-input](https://github.com/marmelab/aor-dependent-input) addon.
+For all those cases, you can use the [`<FormDataConsumer>`](https://marmelab.com/react-admin/Inputs.html#linking-two-inputs) component.
 
 ## UI in production build is empty or broke
 

